src/providers/currency/coin.ts

```diff
@@ -98,7 +98,7 @@ export const availableCoins: CoinsMap<CoinOpts> = {
     paymentInfo: {
       paymentCode: 'EIP681',
       protocolPrefix: { livenet: 'ducatusx', testnet: 'ducatusx' },
-      ratesApi: 'https://bitpay.com/api/rates',
+      ratesApi: 'https://rates.ducatuscoins.com/api/v1/rates/',
```

Comment:
Надо изменить на получение через ApiProvider
```suggestion
      ratesApi: apiProvider.getAddresses().ratesApi,
```